src/routers/MultiRouteRouter.js

```diff
@@ -34,17 +35,22 @@ class MultiRouteRouter extends ApplicationRouter<Array<Object>> {
       return [];
     }
 
-    // multiple destinations given
-    const routes = destinationNames.map((destinationName, index) => RouteUtils.getRoutes(
+    // get array of routes for each destination
+    const multiRoutes = destinationNames.map((destinationName, index) => RouteUtils.getRoutes(
       destinationName,
       end[index],
       start,
       departureTimeQuery,
       false,
     ));
 
-    return Promise.all(routes).then(val => val).catch((err) => {
-      throw LogUtils.logErr(err, routes, 'Could not get all specified routes');
+    // return the best route for each destination
+    return Promise.all(multiRoutes).then((val) => {
+      const bestRoutes = val.map(routes => ((routes.length > 0) ? routes[0] : null));
```

Comment:
maybe just put the ternary on a separate line? 

```
const bestRoutes = val.map((routes) => {
    (routes.length > 0) ? routes[0] : null;
})

Reply:
^ without the braces
```
const bestRoutes = val.map(routes => 
    routes.length > 0) ? routes[0] : null);
```

Reply:
though personally i prefer the original one liner

Reply:
@meganle can you decide

Reply:
I prefer the one liner, but do we need parentheses around `routes.length > 0`?